### DISCUSSION TITLE
很好，越来越想自己手搓一个绳网出来了

### DISCUSSION THREAD

**Discussion by Hanabiiiiii** [share121/inter-knot#1001] [其他]
最近准备学学go，后续想看看能不能用go手搓一个有后端的绳网出来，顺便拿来水毕设
![AC1775730B09D925E3F149CE94AE9B8C](https://github.com/user-attachments/assets/f59abcdd-a305-448f-a2e7-cb50406d9155)

**Comment by Mayf1ower** [comment]
支持！写了后端再把up的这个前端拿过去，再租个服务器部署一下 我保准天天上去冲浪🤓

**Reply by Hanabiiiiii** [reply]
哈哈，我加油，我努力，我哼哼卷，争取明年这个时候能实现基本功能

**Comment by yuhaoNEG** [comment]
好好好，这个项目我投了！

**Comment by tyhpoi** [comment]
这个项目我王多🐟投了

**Comment by trv3wood** [comment]
这个项目我投了

**Comment by TXGY-INTEL** [comment]
这个项目我投了，最好再搓个客户端出来

**Comment by capacitor1** [comment]
这个项目我投了！
争取成为第二个百度贴吧bushi

**Comment by convant** [comment]
贴吧？我不要看到这个口牙

**Comment by NatsukoYamamura** [comment]
写出来的话，我可以提供一个域名（[interknot.work](https://interknot.work)）和云服务器的[给心心]

**Reply by Hanabiiiiii** [reply]
我记下了朋友，但是域名我提前注册了个top的

**Comment by qiuledadada** [comment]
这项目我投了！

**Comment by su2569** [comment]
这个项目我投了！

**Comment by muhan-972** [comment]
这个项目我投了！

**Comment by LERSIA7013** [comment]
这项目我投了，放个屁股出了踹我

**Comment by zzziloveyou** [comment]
很大胆，担心开发的人会找上你的家门，很刑啊

**Reply by Hanabiiiiii** [reply]
哈？你在说啥

**Reply by lllliufeng** [reply]
这个绳网也是同人作品啊

**Comment by tiancaitu** [comment]
这个项目我投了，一百万够不够？不够再加，够了当我没说

**Comment by qiuxi903** [comment]
到时候都不知道有几个绳网了，现在我知道的都不止3个了

**Reply by Hanabiiiiii** [reply]
看了几个，我个人都不太满意，界面ui也好，用户交互逻辑也好都不太是我想象的那样，自己慢慢搞一个出来吧，当玩玩也好，主要学这么久计算机一个项目没有也说不过去🤓

**Comment by isyuah** [comment]
支持，有后端的话拓展各种功能就容易多了

**Comment by Kuuga2000skills** [comment]
这个项目我投了

**Comment by xuan8767** [comment]
这个项目我投了！

**Comment by greenhoie** [comment]
这个项目我投了

**Comment by ChevalierDeSangreal** [comment]
这个项目我投了！This is myGO!!

**Comment by SnowCandle12138** [comment]
不是哥们、原来还能发动图吗

**Comment by H-7942** [comment]
加油

**Comment by Nahida10271027** [comment]
绳网已经有了，要不就整个罗浮杂俎吧（）

**Comment by QingMingqwq** [comment]
这个项目我投了，希望有客户端

**Comment by Pleasurecruise** [comment]
会全栈，希望能加盟。

**Reply by Hanabiiiiii** [reply]
还在新建文件夹，不过如果可以希望交换一下友链😀

**Comment by yys20071108** [comment]
你好在吗问一下我想和你合作制作一个网页版本的走格子

**Reply by Hanabiiiiii** [reply]
你好，感谢邀请，你可以看看我的个人仓库，如你所见我目前还没有任何独立项目，目前，我一般白天有其他的事情，代码学习都是在晚上自己在网上找课程摸索，以我目前的时间精力和技术水平恐怕很难和你合作，所以很抱歉我不能答应，最后感谢邀请，我会继续加油的，希望未来能一起合作

**Comment by AAAduck** [comment]
这个委托我接了

**Comment by Rain-YYC** [comment]
加油！
